src/core/ProgressBar/ProgressBar.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { useStyles } from './styles';
+import LinearProgress from '@material-ui/core/LinearProgress';
+import { BaseProgressProps } from './base';
+
+interface ProgressBarProps extends BaseProgressProps {
+  value: number;
+  color: string;
+  label: string;
+}
+
+const ProgressBar: React.FC<ProgressBarProps> = ({ value, label, color }) => {
+  const classes = useStyles({ color });
+  return (
+    <div>
+      <label className={classes.label}>
+        {label} : {value}
```

Comment:
According to figma there should be a horizontal `<hr/>` tag or `---` line instead of `:`

Reply:
This comment is still unaddressed